tests/Http/Event/ImportEventRequestHandlerTest.php

```diff
@@ -5909,7 +5909,7 @@ public function it_throws_if_videos_url_is_invalid(): void
         $expectedErrors = [
             new SchemaError(
                 '/videos/0/url',
-                'The string should match pattern: ^http(s?):\/\/(www\.)?((youtube\.com\/watch\?v=([^\/#&?]*))|(vimeo\.com\/([^\/#&?]*))|(youtu\.be\/([^\/#&?]*))|(youtube.com/embed/([^\/#&?]*)))'
+                'The string should match pattern: ^http(s?):\/\/(www\.)?((youtube\.com\/watch\?v=([^\/#&?]*))|(vimeo\.com\/([^\/#&?]*))|(youtu\.be\/([^\/#&?]*))|(youtube.com/embed/([^\/#&?]*))|(youtube.com/shorts/([^\/#&?]*)))'
```

Comment:
@grubolsch, you have made a PR in https://github.com/cultuurnet/apidocs/pull/288/files , but the CI still fails on it, I suspect you still have to update apidocs in composer to the latest version? 😉
I you have doubt, we can look at it toghether?